css/animations_transitions.css

```diff
@@ -46,6 +46,11 @@
   background: #008552;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231652438-e7130ee9-7f2f-4e08-85e9-2c5e02c057f7.png)

- [ ] It would be nice if you make your popup description color follow the Figma template as suggested by the previous reviewer :+1: